html/controls/sk/sk-sheet.js

```diff
@@ -306,13 +288,29 @@ function prepareTripEditor() {
 		tripEditor.append($('<table>')
 				.append($('<tr>')
 						.append($('<td colspan="3">')
-								.append($('<input type="number" min="0">').attr('id', 'score'))
-								.append($('<input type="checkbox">').attr('id', 'afterSP'))
-								.append($('<span>').addClass('Infotext').text('SP in this or prior trip'))))
+								.append($('<input type="number" min="0">').attr('id', 'score').keydown(function(event) {
+									if (event.which == 13) tripEditor.find('#submit').click(); }).change(function() {
+										WS.Set(tripEditor.data('prefix')+'Score', $(this).val());
+									}))
+								.append($('<button>').attr('id', 'afterSP').text('SP in this or prior trip').button().click(function() {
```

Comment:
This state isn't propagating correctly on subsequent trips in the UI

Reply:
What does it do and what did you expect?

Reply:
If I click on a subsequent trip I expect the SP button to be green, but it isn't.